stream.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Stream
  * Plugin URI: https://xwp.co/work/stream/
  * Description: Stream tracks logged-in user activity so you can monitor every change made on your WordPress site in beautifully organized detail. All activity is organized by context, action and IP address for easy filtering. Developers can extend Stream with custom connectors to log any kind of action.
- * Version: 3.10.0
+ * Version: 3.10.1
```

Comment:
<!-- simple tasks that must be done -->
**🧹 chore:** As above, this will be done as part of 4.0.1 release.

```suggestion
 * Version: 3.10.0
```